.github/workflows/test-harness.yml

```diff
@@ -0,0 +1,121 @@
+name: Test Harness
+
+on:
+  issue_comment:
+    types: [created]
+
+jobs:
+  test-harness:
+
+    runs-on: ubuntu-latest
+
+    # Only run if we're invoked with a new command comment on a pull request.
+    if: |
+      github.event_name == 'issue_comment' && github.event.action == 'created'
```

Comment:
How about edited too so we can easily rerun tests without creating spam?

Reply:
Test harness creates a comment with the results anyways, so this doesn't completely eliminate the notifications spam. If was solved completely - I'd go for it, but since it doesn't achieve the cover the issue - I think we better not attach any functionality to the edits yet.

I was thinking, in the future, when github actions provide support for it, we could support properly handling edit events to *cancel and then rerun* the test harness invocation initiated by a particular comment. This would work best if you one made a typo in the command, and want to correct it.

Overall, from the general PR discussion flow perspective, I think it'll work better if we have a run per `/test` comment, rather than a single `/test` comment to correspond to multiple runs. The notifications might be a problem, but if they are I'dd address it separately. If we really want an easy way to restart a run - I'd look into doing something with reactions - but it's worth a separate task.